api/chaos/experimentation/v1/experimentation.proto

```diff
@@ -42,7 +42,7 @@ message CreateExperimentResponse {
 }
 
 message GetExperimentsRequest {
-  repeated uint64 ids = 1;
+  string config_type = 1;
 }
```

Comment:
how do we plan to expand usage of this function in the future? isn't fetching specific IDs also a valid use case? or was this already replaced by `GetExperimentRunDetailsRequest`?

Reply:
>how do we plan to expand usage of this function in the future?

We are planning to add an option to ask `GetExperimentsRequest ` to return only running/scheduled/completed tests. I'm currently working on the PR for this.

>isn't fetching specific IDs also a valid use case? or was this already replaced by GetExperimentRunDetailsRequest?

For now, `GetExperimentRunDetailsRequest`  replaced `GetExperimentsRequests` when it comes to getting experiments by their ID. It's possible that we will need to re-add `ids` as part of the `GetExperimentsRequest` but that's not something that we are planning on doing in the near future.